src/components/TimelineComponent.tsx

```diff
@@ -223,7 +242,7 @@ export default class TimelineComponent
         return {
           xColumn: val,
         };
-      }, async () => await this.sortData(val));
+      }, () => this.sortData(val));
```

Comment:
I think that this call should still be awaited, otherwise, resetTimeline will get called before before sorting has completed and result in data being displayed incorrectly.